### PR TITLE
MC-38903: Add more graphic example how new image resizing works

### PR DESCRIPTION
**Pull request by shiftedreality**
The purpose of this PR is to enhance the image resizing content for the Remote storage topics.
Added the Configure caching topic.

https://jira.corp.magento.com/browse/MC-38903

### CODE REVIEW

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+group: configuration-guide
+title: Configure caching for Remote Storage
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+
+Remote storage provide a cache layer for external API calls. The caching is based on [Flysytem cache] provided adapters.
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md:10]:
```suggestion
Remote storage provides a cache layer for external API calls. The caching is based on [Flysystem cache] provided adapters.
```

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Memory cache
+
+Memory cache is enabled by default and provides per-request cache.
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md:14]:
```suggestion
Memory cache is enabled by default and provides per request cache.
```

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Redis cache
+
+A persistent cache stored in Redis and requires an active Redis service and additional configuration
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md:18]:
```suggestion
A persistent cache stored in Redis that requires an active Redis service and additional configuration
```

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+### Redis cache with predis extension.
+
+{:.procedure}
+To use Redis cache with [predis/predis] extension, perform next steps:
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md:23]:
```suggestion
To use Redis cache with [predis/predis] extension, perform the next steps:
```

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Add [predis/predis] extension to your `composer.json`
+
+```bash
+composer require predis/predis
+```
+
+2. Add a configuration to `app/etc/env.php` file:
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md:31]:
```suggestion
2. Add a configuration to the `app/etc/env.php` file:
```

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+                    'scheme' => 'tcp', # Required scheme
+                    'host' => 'redis.internal', # Required hostname of Redis service
+                    'port' => 6379 # Required port of Redis service
+                ]
+        ]
+    ]
+];
+```
+
+[Flysytem cache]: https://flysystem.thephpleague.com/v1/docs/advanced/caching
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-caching.md:49]:
```suggestion
[Flysystem cache]: https://flysystem.thephpleague.com/v1/docs/advanced/caching
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please also add a description of the MC-38903 ticket to this PR as non-Adobe reviewers have no access to this JIRA

**Comment by hguthrie** [comment]
I have made some significant edits to the remote storage content, which has already been rebased with 2.4.2-develop. I will take your new content from this PR and roll it into that branch. This PR will be closed when that is complete.

**Pull request closed without merging**